src/modules/Popup/lib/createReferenceProxy.js

```diff
@@ -7,7 +7,7 @@ class ReferenceProxy {
   }
 
   getBoundingClientRect() {
-    return _.invoke(this.ref.current, 'getBoundingClientRect', {})
+    return _.invoke(this.ref.current, 'getBoundingClientRect') || {}
```

Comment:
This is my the most common mistake with Lodash: `_.invoke()` as 3rd parameter has a call argument, not a default value 🤦